parsers/src/test/scala/uk/gov/ons/addressIndex/parsers/AddressParserMainTest.scala

```diff
@@ -0,0 +1,132 @@
+package uk.gov.ons.addressIndex.parsers
+
+import java.io.File
+import uk.gov.ons.addressIndex.crfscala.CrfScalaJni
+import scala.util.Try
+import scala.xml.XML
+
+//Forgive me father for I have sinned.
+object AddressParserMainTest extends App {
```

Comment:
Because it needs a lot of refactoring, but maybe it will be removed shortly, so refactoring it will be pointless